sw/device/silicon_creator/lib/cert/cwt_dice_chain_entry_payload.hjson

```diff
@@ -0,0 +1,105 @@
+// Copyright lowRISC contributors (OpenTitan project).
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+    name: "cwt_dice_chain_entry_payload",
+
+    variables: {
+        issuer: {
+            type: "string",
+            max-size: 64,
+        },
+        subject: {
+            type: "string",
+            max-size: 64,
+        },
+        code_hash: {
+            type: "byte-array",
+            exact-size: 32,
+        },
+        config_hash: {
+            type: "byte-array",
+            exact-size: 32,
+        },
+        config_desc: {
+            type: "byte-array",
+            max-size: 64,
+        },
+        auth_hash: {
+            type: "byte-array",
+            exact-size: 32,
+        },
+        mode: {
+            type: "byte-array",
+            exact-size: 1,
+        },
+        subject_pk: {
+            type: "byte-array",
+            exact-size: 77,
+        },
+        profile_name: {
+            type: "string",
+            exact-size: 10,
+        },
+    },
+
+    constants: {
+        issuer_label: {
+            type: "integer",
+            value: 1
+        },
+        subject_label: {
+            type: "integer",
+            value: 2
+        },
+        profile_name_label: {
+            type: "integer",
+            value: -4670554
+        },
+        subject_pk_label: {
+            type: "integer",
+            value: -4670552
+        },
+        key_usage_label: {
+            type: "integer",
+            value: -4670553
+        },
+        key_usage_value: {
+            type: "byte-array",
+            value: "20"
+        },
+        code_hash_label: {
+            type: "integer",
+            value: -4670545
+        },
+        config_hash_label: {
+            type: "integer",
+            value: -4670547
+        },
+        config_desc_label: {
+            type: "integer",
+            value: -4670548
+        },
+        auth_hash_label: {
+            type: "integer",
+            value: -4670549
+        },
+        mode_label: {
+            type: "integer",
+            value: -4670551
+        },
+    },
+
+    structure: {
+        issuer_label: issuer
+        subject_label: subject
+        code_hash_label: code_hash
```

Comment:
per RFC, are we also missing `code_descriptor`? https://docs.google.com/document/d/1zBudM3YyasCoYhchRDLJUKKDEtlCQ4Yc5dx9HEE45HU/edit?tab=t.0#bookmark=id.1lf8l9lutgwe

Or should the RFC be updated?

Reply:
From the following cddl used by Android RKP, the `code_descriptor` is optional.
https://cs.android.com/android/platform/superproject/main/+/main:hardware/interfaces/security/rkp/aidl/android/hardware/security/keymint/generateCertificateRequestV2.cddl;l=96

Reply:
got it, we can update the RFC then.